catalog/models.py

```diff
@@ -77,6 +77,14 @@ def get_by_category(self, category: models.Model, ordering: [str]=None) -> model
 
         return self.filter(category__in=categories).order_by(*ordering)
 
+    def get_category_descendants(self, category, ordering):
+        """Return products with prefetch pages and images."""
+        return (
+            self.prefetch_related('page__images')
+            .select_related('page')
+            .get_by_category(category, ordering=ordering)
```

Comment:
It will be work in case when `ordering=None`?

Reply:
@artemiy312 , yes, it will.
`catalog.models.ProductQuerySet#get_by_category` has this code string: `ordering = ordering or ['name']`